cmd/kube-apiserver/app/server.go

```diff
@@ -106,8 +106,11 @@ func RunServer(config *master.Config, sharedInformers informers.SharedInformerFa
 	if err != nil {
 		return err
 	}
+	m.GenericAPIServer.AddPostStartHook("start-k8s.io-informers", func(context genericapiserver.PostStartHookContext) error {
```

Comment:
I'm thinking "k8s.io" is too broad. Maybe this should be kube-apiserver instead?